### DISCUSSION TITLE
How do I customise the blue 'angle' at the very start of the prompt?

### DISCUSSION THREAD

**Discussion by jschuur** [starship/starship#3791] [Q&A] [answered]
I get a blue [symbol](https://www.compart.com/en/unicode/U+276F) at the very start of the prompt, but can't seem to figure out if represents something or is just a style choice.

How can I get rid/modify it? Honestly, the fact that it's so close to the edge of my screen with no padding kind of irks me :)

It doesn't seem to be anything from any of the modules used in the default `format`, because if I turn on `username`, it's still to the left of it, and that seems to be the [first one](https://starship.rs/config/#prompt) in the default prompt template.

**Comment by matchai** [comment, marked as answer]
From the way you describe it, it sounds like it might be the iTerm2's "Marks" feature.

It can be disabled by going to Preferences > Profiles > (your profile) > Terminal, scroll down to "Shell Integration", and turn off "Show mark indicators".

**Comment by jschuur** [comment]
Huh. Thanks @matchai that was it! This explains why I couldn't find if when I poked around in the code too.
